server/src/migrations/20190315183022-create-books.js

```diff
@@ -1,10 +1,9 @@
 module.exports = {
   up: (queryInterface, Sequelize) => queryInterface.createTable('books', {
     id: {
-      allowNull: false,
```

Comment:
I know we are testing for the id before it hits the DB but don't you think this it is safer for the DB just in case. If you feel indifferent that fine.

Reply:
Since the id is a primary key, it does not need to be set to null because it cannot be null. That was why I removed it but then we can add it back